decidim-admin/config/locales/ca.yml

```diff
@@ -636,4 +636,4 @@ ca:
   time:
     formats:
       datepicker: "%m/%d/%Y"
-      timepicker: "%d/%m/%Y %H:%M"
\ No newline at end of file
+      timepicker: "%d/%m/%Y %H:%M"
```

Comment:
Yeah, I was hoping that the git config added to `.gitattributes` would actually enforce that crowdin actually commits the correct final new line. But I don't think that's the case after all.

I'm not sure this patch is worth at the moment since we have no way to enforce the consistency in the future. If you still see value, I can remove the changes to both `.gitattributes` and `config/locales`, otherwise just close.